### PR TITLE
log on start

### PR DESCRIPTION
**Pull request by itzfoxl**
cause we need a indicator to show it started

### CODE REVIEW

File: src/main/java/com/github/dirtpowered/dirtmv/DirtMultiVersion.java [hunk 1]
```diff
@@ -62,6 +63,7 @@ private DirtMultiVersion() {
 
         sharedRandom = new Random();
 
+        System.out.println("(DirtMultiVersion) Started on IP: " + Constants.REMOTE_HOST + ":" + Constants.REMOTE_PORT);
```
Comment (jsdevv) [inline src/main/java/com/github/dirtpowered/dirtmv/DirtMultiVersion.java:66]:
```suggestion
        System.out.println("(DirtMultiVersion) Started on IP: " + Configuration.proxyLocalAddress + ":" + Configuration.proxyLocalPort);
```

File: src/main/java/com/github/dirtpowered/dirtmv/DirtMultiVersion.java [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 package com.github.dirtpowered.dirtmv;
 
+import com.github.dirtpowered.dirtmv.data.Constants;
```
Comment (jsdevv) [inline src/main/java/com/github/dirtpowered/dirtmv/DirtMultiVersion.java:25]:
```suggestion

```

**Review by jsdevv — CHANGES_REQUESTED** [review]
ok

**Comment by itzfoxl** [comment]
i did the changes @MarcinDevelopment

**Comment by itzfoxl** [comment]
ill just close this for now

**Pull request closed without merging**